docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -923,3 +923,14 @@ The current state of compatibility is shown in the following table:
 |`addExtension`|❌
 |`removeExtension`|❌
 |====
+
+== Dependency resolver options
+
+=== Quarkus component variants
+
+`disableQuarkusComponentVariants` project property was introduced in Quarkus 3.26.0 to control whether Quarkus component variants should be added to certain dependencies of a project to enable link:conditional-extension-dependencies.adoc[Quarkus conditional] and build time (deployment) dependencies. Quarkus component variants are enabled by default since 3.26.0.
```

Comment:
'''
link:conditional-extension-dependencies.adoc[Quarkus conditional]
'''
doesn't appear to work in the preview